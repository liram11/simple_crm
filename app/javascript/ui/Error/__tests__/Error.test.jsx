/**
 * @jest-environment jsdom
 */

import React from "react";
import renderer from "react-test-renderer";
import { Error } from "..";

// eslint-disable-next-line no-undef
it("renders correctly", () => {
  const tree = renderer
    .create(<Error message="Some error" />)
    .toJSON();
  // eslint-disable-next-line no-undef
  expect(tree).toMatchSnapshot();
});

// eslint-disable-next-line no-undef
it("does not renders if message is empty", () => {
  const tree = renderer
    .create(<Error message="" />)
    .toJSON();
  // eslint-disable-next-line no-undef
  expect(tree).toMatchSnapshot();
});
