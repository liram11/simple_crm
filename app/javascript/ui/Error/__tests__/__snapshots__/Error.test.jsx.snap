// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does not renders if message is empty 1`] = `null`;

exports[`renders correctly 1`] = `
<div>
  <div
    className="alert alert-danger"
    role="alert"
  >
    Some error
  </div>
</div>
`;
